src/coreComponents/physicsSolvers/fluidFlow/wells/WellSolverBase.cpp

```diff
@@ -56,7 +56,8 @@ WellSolverBase::WellSolverBase( string const & name,
   this->registerWrapper( viewKeyStruct::writeCSVFlagString(), &m_writeCSV ).
     setApplyDefaultValue( 0 ).
     setInputFlag( dataRepository::InputFlags::OPTIONAL ).
-    setDescription( "When set to 1, write the rates into a CSV file" );
+    setDescription( "When set to 1, write the rates into a CSV file"
+                    "Set to 0 by default" );
```

Comment:
```suggestion
);
```
The default gets published with the doc. So no need to add that to the description.